man/coredump.conf.xml

```diff
@@ -119,6 +119,22 @@
         either value to 0 to turn off size-based
         clean-up.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+         <term><varname>RateLimitInterval=</varname></term>
+         <term><varname>RateLimitBurst=</varname></term>
+         <listitem><para>Configures the rate-limiting that is applied to
+         core dump generated per daemon. Whenever a daemon crashes need to
+         compare the number of core dump files present in the core dump path
+         belonging to that daemon and generated within the ratelimitinterval
+         against the ratelimitburst. If the ratelimitburst is reached, the
```

Comment:
"ratelimitinterval" → "interval time" and "ratelimitburst" → "burst value"

Reply:
I have addressed the review comment. Please check once